hoextdown/hoextdown.dict

```diff
@@ -0,0 +1,49 @@
+asterisk="*"
```

Comment:
Just FYI: 
the names of the tokens are redundant and we support them for compatibility with AFL's syntax. 
You may write
```
"*"
```

instead of 
```
asterisk="*"
```

But this way will work too.